<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetsImportTableDialog</class>
 <widget class="QDialog" name="WidgetsImportTableDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="WidgetsImportView" name="ImportView" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="BtnReplace">
       <property name="text">
        <string>REPLACE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnInsert">
       <property name="text">
        <string>INSERT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnCancel">
       <property name="text">
        <string>CANCEL</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetsImportView</class>
   <extends>QWidget</extends>
   <header>WidgetsModule/internal.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
